_examples/main.go

```diff
@@ -73,8 +74,13 @@ func main() {
 		panic(fmt.Sprintf("Could not connect to RabbitMQ: %v", err.Error()))
 	}
 
+	r := bytes.NewReader(data)
+	s, err := jazz.DecodeYaml(r)
```

Comment:
what is s? What is r?

Reply:
r is io.reader, this way it's more generic and object with function `func Read(p []byte) (n int, err error)` that reads it's content to buffer p. In this case r is reader which reads from byte buffer, but it could also be for example reader from file.
s is structure with description of all exchanges and queues which will be created or deleted by subsequent calls to `CreateScheme()` and `DeleteScheme()`

Reply:
I was just wondering if it woudn't be more clear to make a full name variable - reader, structure? One letter variable can be very counfusing if there is not some language specific naming.

Reply:
Yeah, it should be more clear now.